src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/RespondentLiPResponse.java

```diff
@@ -11,6 +12,10 @@
 @AllArgsConstructor
 public class RespondentLiPResponse {
 
+    @JsonUnwrapped
+    private PartAdmitResponseLiP partAdmitResponseLiP;
```

Comment:
I think this field should have related CCD field, could you please check?

Reply:
Related field should be the following - specDefenceAdmittedRequired

Reply:
Agree with Kenneth - You need to change it also in CUI and CCD prs

Reply:
<img width="907" alt="partAdmitCUICCD" src="https://user-images.githubusercontent.com/25331371/227480016-01cfe8f8-28fd-443a-bfd4-cc4500d1b445.PNG">